src/main/telemetry/crsf.c

```diff
@@ -388,6 +391,12 @@ void crsfSendMspResponse(uint8_t *payload)
 static void processCrsf(void)
 {
     static uint8_t crsfScheduleIndex = 0;
+#ifdef USE_GPS
+    if (!gps_telemetry_added && (featureIsEnabled(FEATURE_GPS) || sensors(SENSOR_GPS))) {
```

Comment:
Isn't it possible to keep this only in initialization function?
If it must be dynamic, sensor removal should be handled too

This code is duplicated below,, its should be extracted into function

Also is OR between FEATURE_GPS and SENSOR_GPS correct? (looks suspicious)

Reply:
You are right. I fixed this first and then the configuration issue afterwards. Should have done it the other way round. Oh well.
Should work with original code due to the fix in config.c